crontabber/tests/test_crontabber.py

```diff
@@ -330,6 +370,11 @@ def test_pop(self):
         popped = self.database.pop('foo', 'default')
         eq_(popped, 'default')
         assert_raises(KeyError, self.database.pop, 'bar')
+        try:
+            del self.database['bar']
+            assert False, "it should have raise a KeyError on del"
+        except KeyError:
+            pass
```

Comment:
No... they're not similar. I'm pretty sure `assert_raises` can't work as a context manager. I don't know if nose has something else that does work as a context manager. It's definitely worth switching to a context manager form.

```
with self.assertRaises(KeyError):
    del self.database['bar']
```